helpers/hllm.py

```diff
@@ -265,8 +265,33 @@ def _call_api_sync(
         temperature=temperature,
         **create_kwargs,
     )
-    model_response = completion.choices[0].message.content
-    return model_response, completion
+    # Calculate the cost.
+    models_info_file = ""
```

Comment:
I've proposed in a different PR to pass the LlmCost object to get_completion but this will make caching difficult.
You can file a bug for this issue and we can think about how to do it